### DISCUSSION TITLE
Can I use dielectric constants specific to the type of residue with gmx_MMPBSA?

### DISCUSSION THREAD

**Discussion by narubi2** [Valdes-Tresanco-MS/gmx_MMPBSA#474] [Q&A] [answered]
Hi all.

I found some articles suggesting to use residue type specific dielectric constants instead of one dielectric constant for all residues (https://pubs.acs.org/doi/10.1021/acs.jcim.8b00248).
The effect of using residue type specific dielectric constants might depend on the type of protein or ligand. But I think it is worth a try.

Any suggestions or advice would be greatly appreciated.

**Comment by marioernestovaldes** [comment, marked as answer]
you can adjust the dielectric constant based on residues with the cas_intdiel variable:

https://valdes-tresanco-ms.github.io/gmx_MMPBSA/dev/input_file/#alanine_scanning-namelist-variables

**Reply by narubi2** [reply]
Oh, I didn't know 'cas_intdiel' variable.
Thanks you @marioernestovaldes
